### ISSUE TITLE
ObO: fetch data from Airtable

### ISSUE THREAD

**Issue by King-Mob** [SolidarityEconomyAssociation/open-data#78]

**Comment by ColmMassey** [comment]
Do we think we can get something on the map next week? Are you waiting for any actions from Anna?

**Comment by ColmMassey** [comment]
I am seeing Airtable password resets. Do you have the access you need? @King-Mob ?

**Comment by King-Mob** [comment]
I think that's probably Nick. I had to pass the airtable task to him in order to get it done this week.

**Comment by wu-lee** [comment]
Yep, it was me - I've created an Airtable account with read-only access so that any API key used in the configs (and so committed in the source code in the download URL) will not be able to do anything destructive. The account uses the SEA tech.accounts email address.
